connectivity/FEATURE_BLE/source/cordio/source/PalGapImpl.cpp

```diff
@@ -173,6 +207,12 @@ ble_error_t PalGap::set_scan_response_data(
     const advertising_data_t &scan_response_data
 )
 {
+    tr_info("[PAL] Set scan response data - "
```

Comment:
I thought maybe it would useful for the user to see the chain of function calls and the parameters being passed between them. Like what if the PAL trace reported an advertising_channel_map other than ALL_ADVERTISING_CHANNELS. The fact that a corresponding trace is observed from PAL all is a good indication that the GAP function succeeded without error. In saying that I can see the redundancy and the overhead that comes with adding "echo" traces in PAL

Reply:
if there is a meaningful conversion or parameter checking - you can trace that instead.